app/src/main/res/layout/fragment_button_primary.xml

```diff
@@ -57,4 +57,16 @@
     app:buttonIcon="@drawable/bpk_long_arrow_right"
     app:buttonType="primary"/>
 
+  <net.skyscanner.backpack.button.BpkButton
```

Comment:
Can we please move this to the `Icon Alignment` example screen?

Reply:
👍